### PR TITLE
增加Lua函数“Duel.GetRandomNumber()”

### PR DESCRIPTION
**Pull request by YSPplayer**
●void Duel.GetRandomNumber()
int Duel.GetRandomNumber(int min,int max)
返回min和max之间的1个随机数(包括二者自身)，min最小为0。

### CODE REVIEW

**Comment by purerosefallen** [comment]
LGTM

**Comment by purerosefallen** [comment]
But please remove all `////zdiy////` blocks.

File: libduel.cpp [hunk 1]
```diff
@@ -13,6 +13,18 @@
 #include "group.h"
 #include "ocgapi.h"
 
+/////zdiy/////
```
Comment (purerosefallen) [inline libduel.cpp]: Remove those blocks.

File: libduel.cpp [hunk 2]
```diff
@@ -4792,6 +4804,9 @@ int32 scriptlib::duel_majestic_copy(lua_State *L) {
 }
 
 static const struct luaL_Reg duellib[] = {
+	/////zdiy/////
+	{ "GetRandomNumber", scriptlib::duel_get_random_number },
```
Comment (purerosefallen) [inline libduel.cpp]: Put this line at the bottom of extra function block, right below `SetSummonCancelable`

File: scriptlib.h [hunk 3]
```diff
@@ -398,6 +398,9 @@ class scriptlib {
 	static void open_grouplib(lua_State *L);
 
 	//Duel functions
+	/////zdiy/////
+	static int32 duel_get_random_number(lua_State* L);
```
Comment (purerosefallen) [inline scriptlib.h]: Put this line below `static int32 duel_set_summon_cancelable(lua_State *L);`

**Review by purerosefallen — CHANGES_REQUESTED** [review]

**Comment by YSPplayer** [comment]
格式已修改

**Comment by IMJoyJ** [comment]
Function implemented and this pr should be closed now.

**Pull request closed without merging**